_security/configuration/generate-certificates.md

```diff
@@ -207,11 +207,13 @@ Then copy and paste the output into `opensearch.yml`.
 This process generates many files, but these are the ones you need to add to each node:
 
 - `root-ca.pem`
-- `admin.pem`
-- `admin-key.pem`
+- (Optional) `admin.pem`
+- (Optional) `admin-key.pem`
 - (Optional) `node1.pem`
 - (Optional) `node1-key.pem`
 
+For most users, the `admin.pem` and `admin-key.pem` files only need to be added to nodes you plan to run the `securityadmin` script or reload certificates from. After configuring your certificates and starting OpenSearch, you can run `securityadmin.sh` to initialize the Security plugin. For information about how to use this script, see [Applying changes to configuration files]({{site.url}}{{site.baseurl}}/security/configuration/security-admin/). If you intend to run the `securityadmin` script directly from a node, it will need to have a copy of `admin.pem` and `admin-key.pem` on it.
```

Comment:
The security plugin will be seeded by default from the custom yml files in `config/opensearch_security/`. (`internal_users.yml`, `roles.yml`, roles_mapping.yml`, etc.). The `securityadmin.sh` tool is used for making changes on a running cluster.

Reply:
Yeah this was already here, but I agree it is not clear. Let me remove it.